apps/studio/components/interfaces/Database/Wrappers/WrappersDisabledState.tsx

```diff
@@ -24,34 +33,46 @@ const WrappersDisabledState = () => {
     'extensions'
   )
 
+  const { mutateAsync: enableExtension } = useDatabaseExtensionEnableMutation()
+
   const onEnableWrappers = async () => {
     if (wrappersExtension === undefined || vaultExtension === undefined) return
+    if (project === undefined) return console.error('Project is required')
+    if (project.connectionString === undefined)
+      return console.error('Connection string is required')
+
     setIsEnabling(true)
 
     const requiredExtensions = await Promise.all([
-      await meta.extensions.create({
+      await enableExtension({
```

Comment:
`Promise.all()` isn't doing anything here because of double `await`s
```suggestion
      enableExtension({
```